src/services/test-result.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { IResponse } from './../interfaces/interface';
+import TestResultService from "./test-result";
+import { DynamoDB } from "aws-sdk";
+
+jest.mock('aws-sdk');
+
+const mockTableName = 'MockTableName'
+const mockDBWithPutSuccess = jest.fn<DynamoDB.DocumentClient>(() => ({
+	put: jest.fn((params, closure) => closure(null)),
+}))();
+const mockDBWithPutFailure = jest.fn<DynamoDB.DocumentClient>(() => ({
+	put: jest.fn((param, closure) => closure({})),
+}))();
+const validTestResult = {
+	_candidateId: "some candidate Id",
```

Comment:
Why _ here?

Reply:
because in getJournal candidate id is stored in _candidateId and I would like to be consistent. We could change it in the future if we want